src/config/locales/en-US.json

```diff
@@ -340,7 +340,8 @@
     "done":"DONE",
     "draft_save_title":"Saving Draft",
     "draft_update":"Update current draft",
-    "draft_save_new":"Save as new draft"
+    "draft_save_new":"Save as new draft",
+    "load_draft_tooltip": "Load your lastest draft here"
```

Comment:
Let's introduce a separate section where all walkthrough related texts reside so it's easier to manage them in one place later...? moving forward we can even devise some mechanism to pair them with walkthrough ids, but not now.

```
"walkthrough":{
   "load_draft":"asdfasldkf"
}
```